adr/0025-composition.md

```diff
@@ -0,0 +1,282 @@
+# 25. Practical Component Composition
+
+Date: 2024-07-02
+
+## Status
+
+Draft
+
+## Context
+
+Presently "composition" within Zarf is only possible at the package level. This can only be done with a
```

Comment:
It can also be done locally with path: ../ but agree the skeleton is the more obtuse use case (and has been acknowledged in the past)

Reply:
(skeleton is only used in the OCI case in Zarf)

Reply:
I thought both local path and OCI imports were considered "skeleton packages". They are the same in all respects, correct?